blazor/rich-text-editor/toolbar.md

```diff
@@ -9,18 +9,18 @@ documentation: ug
 
 # Toolbar types in Blazor RichTextEditor
 
-The Rich Text Editor toolbar contains a collection of tools such as bold, Italic, and text alignment buttons that are used to format the content. However, in most integrations, you can customize the toolbar configurations easily to suit your needs. 
+The Rich Text Editor toolbar contains a collection of tools such as bold, Italic, and text alignment buttons that are used to format the content.
 
-![Toolbar](./images/blazor-richtexteditor-toolbar-types.png)
+![Blazor RichTextEditor toolbar](./images/blazor-richtexteditor-toolbar-types.png)
 
-The editor allows to configure different types of toolbar using [RichTextEditorToolbarSettings.Type](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.RichTextEditorToolbarSettings.html#Syncfusion_Blazor_RichTextEditor_RichTextEditorToolbarSettings_Type) property. The types of toolbar are:
+You can customize the toolbar configurations by usig the [RichTextEditorToolbarSettings.Type](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.RichTextEditorToolbarSettings.html#Syncfusion_Blazor_RichTextEditor_RichTextEditorToolbarSettings_Type) property. The types of toolbar are:
```

Comment:
scrollable mode?

Reply:
Expand and Multi-row only mentioned in outline sir.

<img width="387" alt="image" src="https://user-images.githubusercontent.com/93525217/201016069-01801090-7b59-41ce-93b1-6960e4520ec2.png">

Reply:
Added sir.